apps/test-enc-abs-pwm/main.cpp

```diff
@@ -23,7 +23,11 @@ DigitalOut led(LED1);
 int main() {
 
     float inverter = +1;
+<<<<<<< HEAD
```

Comment:
fix these merge conflict artifacts lol

Reply:
I rebased with alex's arm code at some point early on (I think), so thats why I have all this added. Not actually my code.